xenonpy/descriptor/base.py

```diff
@@ -266,7 +266,7 @@ def transform(self, entries: Sequence, *, return_type=None, target_col=None, **k
             entries = entries[target_col]
 
         # Special case: Empty list
-        if len(entries) is 0:
+        if not entries:
```

Comment:
`not` condition in python means an empty container or `None` type.
Here, when we write `len() == 0`, it gives us a clear image that the container is empty. since `entries` can no be `None`, so I suggest to rewrite this to 'len(entries) == 0'.